packages/yoroi-extension/app/connector/components/signin/cardano/ConnectionInfoComponent.js

```diff
@@ -0,0 +1,48 @@
+import { Box, Typography } from '@mui/material';
```

Comment:
the `// @flow` flag should be at the top of every file 
```suggestion
// @flow 
import { Box, Typography } from '@mui/material';
```